src/tools/aml-language-server/tsconfig.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
What's the motivation for having a different ts config? Do you need more relaxed one?